test/command/test_fluentd.rb

```diff
@@ -895,6 +896,27 @@ def multi_workers_ready?
       )
     end
 
+    # https://github.com/fluent/fluentd/issues/2915
+    test "ruby path contains spaces" do
+      conf = <<CONF
+<source>
+  @type dummy
+  tag dummy
+</source>
+<match>
+  @type null
+</match>
+CONF
+      ruby_path = ServerEngine.ruby_bin_path
+      tmp_ruby_path = File.join(TMP_DIR, "ruby with spaces")
```

Comment:
Using symlink to conf file is enough? if so, [this change](https://github.com/fluent/fluentd/pull/2920/commits/ce691c0f20786f4d31c85a88f2b87a8292e6ffbd#diff-dbe0e1ec4079138e48ca6a4d7c7248f9R885 ) is unnecessary.

Reply:
> Using symlink to conf file is enough?

Cannot test it by this way. [Testing `RUBYOPT`](https://github.com/fluent/fluentd/blob/ce691c0f20786f4d31c85a88f2b87a8292e6ffbd/lib/fluent/supervisor.rb#L904) doesn't use fluentd options, it added at [here](https://github.com/fluent/fluentd/blob/ce691c0f20786f4d31c85a88f2b87a8292e6ffbd/lib/fluent/supervisor.rb#L911).
I'm considering how to test without modifying supervisor, but currently I have no idea.

Reply:
I see. How about mocking it, then? like below.

```rb
mock(ServerEngine). ruby_bin_path { File.join(TMP_DIR, "ruby with spaces") }
```

Reply:
Ah, I see.

Reply:
Oops, it can't mock since it executed in [another process](https://github.com/fluent/fluentd/blob/ce691c0f20786f4d31c85a88f2b87a8292e6ffbd/test/command/test_fluentd.rb#L105).

Reply:
Right... it's okay to leave as it is. 
I think it's better to change `FLUENTD_RUBY_PATH` to another name to tell developers that this ENV var is **only** for tests.

Reply:
fixed